WeightProducer/python/MCSampleValues.py

```diff
@@ -84,422 +84,624 @@ class MCSampleValuesHelper():
+                                      kFactor_2018=1.165, kFactorSource_2018="Same as 2017 (2018 NNLO sample not ready)"),
         },
         "DYJetsToLL_M-50_HT-800to1200" : {
-            "CrossSection" : XSValues(XS_2016=0.775392),
-            "kFactor" : kFactorValues(kFactor_2016=1.23),
+            "CrossSection" : XSValues(XS_2016=0.6229, XSSource_2016="XSDB",
+                                      XS_2017=0.8052, XSSource_2017="XSDB",
+                                      XS_2018=0.8052, XSSource_2018="Same as 2017"),
+            "kFactor" : kFactorValues(kFactor_2016=1.23, kFactorSource_2016="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#DY_Z",
+                                      kFactor_2017=1.165, kFactorSource_2017="NNLO/LO=6225.42/5343",
+                                      kFactor_2018=1.165, kFactorSource_2018="Same as 2017 (2018 NNLO sample not ready)"),
         },
         "DYJetsToLL_M-50_HT-1200to2500" : {
-            "CrossSection" : XSValues(XS_2016=0.186222),
-            "kFactor" : kFactorValues(kFactor_2016=1.23),
+            "CrossSection" : XSValues(XS_2016=0.1512, XSSource_2016="XSDB",
+                                      XS_2017=0.1933, XSSource_2017="XSDB",
+                                      XS_2018=0.1933, XSSource_2018="Same as 2017"),
+            "kFactor" : kFactorValues(kFactor_2016=1.23, kFactorSource_2016="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#DY_Z",
+                                      kFactor_2017=1.165, kFactorSource_2017="NNLO/LO=6225.42/5343",
+                                      kFactor_2018=1.165, kFactorSource_2018="Same as 2017 (2018 NNLO sample not ready)"),
         },
         "DYJetsToLL_M-50_HT-2500toInf" : {
-            "CrossSection" : XSValues(XS_2016=0.00438495),
-            "kFactor" : kFactorValues(kFactor_2016=1.23),
+            "CrossSection" : XSValues(XS_2016=0.003659, XSSource_2016="XSDB",
+                                      XS_2017=0.003468, XSSource_2017="XSDB",
+                                      XS_2018=0.003468, XSSource_2018="Same as 2017"),
+            "kFactor" : kFactorValues(kFactor_2016=1.23, kFactorSource_2016="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#DY_Z",
+                                      kFactor_2017=1.165, kFactorSource_2017="NNLO/LO=6225.42/5343",
+                                      kFactor_2018=1.165, kFactorSource_2018="Same as 2017 (2018 NNLO sample not ready)"),
         },
         "DYJetsToLL_M-50" : {
-            "CrossSection" : XSValues(XS_2016=6025.2),
+            "CrossSection" : XSValues(XS_13TeV=6225.42, XSSource_13TeV="XSDB (NNLO)"),
         },
         "ZJetsToNuNu_HT-100To200" : {
-            "CrossSection" : XSValues(XS_2016=344.8305),
-            "kFactor" : kFactorValues(kFactor_2016=1.23),
+            "CrossSection" : XSValues(XS_2016=93.35, XSSource_2016="XSDB",
```

Comment:
this number and the other zinv numbers for 2016 from xsdb seem obviously wrong to me.